src/panel/index.js

```diff
@@ -39,7 +41,7 @@ class Panel extends PureComponent {
             <div className='mdl-card mdl-card--border mdl-shadow--4dp' data-spy={this.spyId} data-focus='panel'>
                 <div className='mdl-card__title mdl-card--border' data-focus='panel-title'>
                     {title && <h3 data-spy-title>{i18next.t(title)}</h3>}
-                    {shouldDisplayActionsTop && <div className='buttons'>{Buttons}</div>}
+                    {shouldDisplayActionsTop && <Buttons editing={editing} toggleEdit={toggleEdit} getUserInput={getUserInput} save={save}/>}
```

Comment:
@Ephrame if you declare Buttons directly a components, you remove the ability to user to define is own actions.